web-dev/src/components/Profile/index.js

```diff
@@ -45,7 +45,7 @@ const ProfileScreen = () => {
                     <ProfileComments />
                 </div>
 
-                {currentUser.isAdmin &&
+                {currentUser && currentUser.isAdmin &&
```

Comment:
@saigonfox Please handle when currentUser is null. This throw error on both current user and other user profile page

Reply:
Got it.